### DISCUSSION TITLE
How bind all m-keys on my asus g15 advantage 2021?

### DISCUSSION THREAD

**Discussion by sanslash332** [seerge/g-helper#3244] [Q&A] [answered]
Hello guys, a small question for one missing key.

On the gHelper configuration window, I see options for bind m1, m2, m3 and m4 keys, and other combinations with the fn key... But m5 is missing.

Is possible bind or add that missing key?
Or that key is controlled outside asus services.

Specifically the key that isn't configurable is the fourth key, lef to right. That key changes the power mode on gHelper and it dettect the change. The most right key is that detected and m4 and the default function is open gHelper /armoury crate.

My notebook is a Asus rog strix g15 advantage edition, 2021.

Thanks so far!

**Comment by seerge** [comment, marked as answer]
@sanslash332 hello,

I was answering this multiple times already https://github.com/seerge/g-helper/issues/2022 https://github.com/seerge/g-helper/issues/2157 

M4 and Fn+F5 emit same keycode on Strix devices. This key is reserved for toggling profiles forward, and Shift + Fn + F5 for toggling profiles backwards.

So It won't be bindable under Extra :)

Thanks

**Reply by sanslash332** [reply]
O, thanks @seerge


Sorry, I didn't found these other questions about te same topic.

Thanks, the doubt was clarified.

Thanks for the project, is simply... awesome.
